Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -148,27 +156,58 @@ export type NotificationProps = {
 };
 
 const makeLinkLabel = (link: NotificationLink) => (
-  <ActionButton css={linkButton} onClick={link.onClick}>
+  <ActionButton styles={linkButton} onClick={link.onClick}>
     {link.label}
   </ActionButton>
 );
 
 const defaultCardContentRenderer = (props: CardProps) => {
-  const { title, description, type, link, links } = props;
+  const { title, description, type, link, links, leftLinks, rightLinks } = props;
+
+  const rightLinkList = rightLinks ?? links ?? [link];
+  const leftLinkList = leftLinks ?? [];
+
+  const stackProps: IStackProps = {
+    horizontal: true,
+    horizontalAlign: 'space-between',
+    tokens: {
+      childrenGap: '20px',
+      padding: '0 16px 0 0',
+      maxHeight: '24px',
+    },
+  };
+
   return (
     <div css={cardContent}>
       {type === 'error' && <Icon css={errorType} iconName="ErrorBadge" />}
       {type === 'success' && <Icon css={successType} iconName="Completed" />}
       {type === 'warning' && <Icon css={warningType} iconName="Warning" />}
       {type === 'question' && <Icon css={questionType} iconName="UnknownSolid" />}
       {type === 'congratulation' && <Icon css={congratulationType} iconName="Trophy2Solid" />}
+      {type === 'custom' && (
+        <Icon
+          css={css`
+            margin-top: ${iconMargin};
+            color: ${props.color ?? SharedColors.gray10};
+          `}
+          iconName={props.icon ?? 'UnknownSolid'}
+        />
+      )}
       <div css={cardDetail}>
         <div css={cardTitle}>{title}</div>
         {description && <div css={cardDescription}>{description}</div>}
-        {link && makeLinkLabel(link)}
-        {links?.map((link) => (
-          <div key={link.label}>{makeLinkLabel(link)}</div>
-        ))}
+        <Stack horizontal horizontalAlign="space-between">
+          <Stack {...stackProps}>
+            {leftLinkList.map(
+              (link) => link != null && <Stack.Item key={link.label}>{makeLinkLabel(link)}</Stack.Item>
+            )}
+          </Stack>
+          <Stack {...stackProps}>
+            {rightLinkList.map(
+              (link) => link != null && <Stack.Item key={link.label}>{makeLinkLabel(link)}</Stack.Item>
```

Comment:
Why would a link be null?

Reply:
I was using that to indicate a gap, to make the design work, but you're right that this is unnecessary now. I'll fix it.